### PR TITLE
Portfolio: f9 Validate contact form.

### PR DESCRIPTION
**Pull request by jcmunav63**
**Description**
For this milestone of your portfolio website, you will implement a client-side validation of the data in the contact form before submission.

**General requirements**
- Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
- Make sure that you used the correct [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md).
- Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

**HTML/CSS & JavaScript requirements**
- Follow our list of [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
- Follow our list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).

**Project requirements**
- You need to implement a simple validation:
   - The content of the email field has to be in lowercase.
- You should implement the following interactions:
   - When the user submits the form, you check if the email is in lowercase.
   - If the validation is OK, the form is sent.
   - If the validation is not OK, you show an error message to the user near the submit button informing them of the error and the form is not sent.
- When you create the error message, you should stick to the design guidelines (e.g., font, colors, margins) of [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

**Work and submission mode**
- You should implement the above requirements in all repositories in your pair-programming group. That means that each group member needs to open a new pull request in their repo.
- We will check the commit history to make sure that everybody has contributed code on each project.
- You should ask for a review and submit this activity individually.

### CODE REVIEW

**Comment by jcmunav63** [comment]
IMPORTANT NOTE:
Please take into consideration that we still have some WEBHINT linters errors in the local and remote repositories. I believe this is a global problem that affects all students while it's not resolved.
f9_ERROR_in_WEBHINT_linters_06-21-2023

File: css/styles.css [hunk 1]
```diff
@@ -4,6 +4,52 @@
...
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  background-color: rgba(39, 4, 197, 0.4);
+}
+
+.modal-content {
+  display: flex;
+  background-color: #fefefe;
+  justify-content: space-around;
```
Comment (Shedrack-Sunday) [inline css/styles.css:23]: - [ ] Kindly do well to remove comented code from your codebase. This is to make sure you don't have a clustered project and codease 👍

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @jcmunav63      👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done extremely well so far in this project,  However, there are some adjustments you need to make to move to the next project. 💯



## Highlights   
- Good use of GitHub features ✔️  
- Nice work with the use of validations✔️ 
- Good project setup ✔️ 
  
# Required Changes  🌵 

- [ ] You made some impressive implementations which I must say is a job well done. However, there are a few  I suggest   you make In line with the requirement

_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing really improves your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by cynthiainga — APPROVED** [review]
## STATUS APPROVED ✔️ 🎉 ✔️ 

Hi @jcmunav63, 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- [ ] Correct GitHub flow ✔️ 
- [ ] Good work with the use of validations ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can get the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**